templates/first-snap/build.html

```diff
@@ -11,7 +11,6 @@
       <div class="col-8">
         <ol class="p-list-step has-margin">
           {% for step in steps %}
-            {% if not loop.last %}
```

Comment:
Surely this was here for a reason?
Do the other flows still work as they did before?

Reply:
It's not clear to me what that reason is. Why skip the last item in the list of actions?

https://github.com/canonical-websites/snapcraft.io/pull/1279#issuecomment-436935795

Everything seems to work fine without it. Can you elaborate on its purpose?

Reply:
Ah, because the last item in the list was a warning, and the loop was making it so it was step "x" instead of being related to the step before.

See the python 'build' step with your change:
![screenshot_2018-12-13 snapcraft - snaps are universal linux packages](https://user-images.githubusercontent.com/479384/49954222-e65c8380-fef7-11e8-8bb8-cd0a0c986125.png)

This should probably be fixed outside of this PR

Reply:
https://github.com/canonical-websites/snapcraft.io/issues/1439 to track this issue